### PR TITLE
.werft: Add honeycomb trace as job result

### LINKED ISSUE #7996: werft: Add Honeycomb trace URL to job results
It would be a lot more convenient if every job would have its own trace URL as a result. 

Right now, to find a trace, it demands knowledge around how to do queries in Honeycomb, and thus its not accessible for everyone

### PR DESCRIPTION
**Pull request by ArthurSens**
## Description

Hmmm, unfortunately, this one doesn't seem to be possible at the moment.

After starting a [job that adds the result](https://werft.gitpod-dev.com/job/gitpod-build-arthursens-werft-add-honeycomb-7996.4/results), the result URL redirects us to [a non-existing trace](https://ui.honeycomb.io/gitpod/datasets/werft/trace/1afedf4da7b133862fde27257bbfa9e7?span=c24c3da57e6f51b6). Since [the trace is queriable using the UI](https://ui.honeycomb.io/gitpod/datasets/werft/result/xQiEs99h58j) we know it was ingested without problems, however, after[ looking into the trace](https://ui.honeycomb.io/gitpod/datasets/werft/result/wdPLAyCjbNC/trace/iuw1mpetckW?span=c24c3da57e6f51b6) we notice that Honeycomb generates a random identifier for each trace, making it impossible to build a stable URL for our jobs :(

## Related Issue(s)

Should have fixed #7996 

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jankoehnlein** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @jankoehnlein` in a comment when ready.

Associated issue: *#7996*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ArthurSens** [comment]
Tagging @gitpod-io/platform here just to make sure I haven't missed anything... Any workarounds we could do to make traces more accessible?

**Comment by ArthurSens** [comment]
Oh, got a nice tip from twitter: https://twitter.com/jessitron/status/1489683062657110021

**Comment by mads-hartmann** [comment]
@ArthurSens Awesome. I really love the idea so if you can get it to work using the tip that's perfect. I'm removing the review request for now - please re-request once you have tried the tip ☺️

File: .werft/util/werft.ts
Comment (ArthurSens) [inline .werft/util/werft.ts:114]: hey @mads-hartmann, let me borrow your typescript knowledge a little bit here

For some reason, this result is being cut after the traceID. `trace_start_ts` and `trace_end_ts` are not being added to the result...?

Because of that, the result does work for some minutes, but after 1~2 hours Honeycomb can't find it anymore. Honeycomb needs `trace_start_ts` and `trace_end_ts` for it to work without time restrictions

Comment (ArthurSens) [reply .werft/util/werft.ts]: Nevermind, just adding double quotes solved the issue :)

**Review by mads-hartmann — APPROVED** [review]
This is so cool 🎉

**Pull request merged**
